docs/user/contacts.md

```diff
@@ -0,0 +1,53 @@
+# Contacts
+
+## Table of Contents
+
+- [Bug Report](#Bug-Report)
+- [Proposal](#Proposal)
+- [Feature Request](#Feature-Request)
+- [Security Issue Report](#Security-Issue-Report)
+
+### Bug Report
+
+A bug is a demonstrable problem which produce incorrect result or to behave in unintended ways.
+Bug reports are helpful for developers who maintain the Vald project.
+
+A good bug report shouldn’t leave others needing to ask you for more information constantly.
```

Comment:
```suggestion
A good bug report should not leave others needing to ask you for more information constantly.
```

Reply:
Thank you for your review.
I will fix it soon!